scripts/validate-dashboards.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+# List of files to exclude
+excludes=( "values.schema.json" "home.json" )
+exculde_string=$(IFS="|"; echo "${excludes[*]}")
```

Comment:
Typo. To be fixed globally.
```suggestion
exclude_string=$(IFS="|"; echo "${excludes[*]}")
```

Reply:
fixed